HOW_TO.md

```diff
@@ -0,0 +1,73 @@
+# Benchmark-runner: How to?
+
+_**Table of Contents**_
+
+<!-- TOC -->
+- [Add new workload to benchmark runner](#add-new-workload-to-benchmark-runner)
+- [Add workload to grafana dashboard](#add-workload-to-grafana-dashboard)
+- [Monitor and debug workload](#monitor-and-debug-workload)
+
+<!-- /TOC -->
+
+## Add new workload to benchmark runner
+1. git clone https://github.com/redhat-performance/benchmark-runner
+2. cd benchmark-runner
+3. Open [benchmark_operator_workloads.py](benchmark_runner/benchmark_operator/benchmark_operator_workloads.py)
+4. Create new `workload method` for Pod and VM under `Workloads` section in [benchmark_operator_workloads.py](benchmark_runner/benchmark_operator/benchmark_operator_workloads.py).
+   It can be duplicated from existing workload method: `def stressng_pod` or `def stressng_vm` and customized workload run steps accordingly
+5. Add workload method name (workload_pod/workload_vm) to environment_variables_dict['workloads'] in [environment_variables.py](benchmark_runner/main/environment_variables.py)
+6. Create workload folder in each workload flavor [func_ci](benchmark_runner/benchmark_operator/workload_flavors/func_ci), [perf_ci](benchmark_runner/benchmark_operator/workload_flavors/perf_ci), [test_ci](benchmark_runner/benchmark_operator/workload_flavors/test_ci)
+   1. For each workload folder:
+      1. Add 'workload_data_template' for configured parameters and configure workload index name 'es_index_name' i.e. [stressng_data_template.yaml](benchmark_runner/benchmark_operator/workload_flavors/func_ci/stressng/stressng_data_template.yaml)
+      2. Add workload Pod and VM CRD template inside [internal_data](benchmark_runner/benchmark_operator/workload_flavors/func_ci/stressng/internal_data)
+7. Add workload folder path in [MANIFEST.in](MANIFEST.in), for each flavor 'func_ci', 'perf_ci', 'test_ci' add 2 paths: the workload path to 'workload_data_template.yaml' and path to 'internal_data' Pod and VM template yaml files
+8. For test and debug workload, need to configure [environment_variables.py](benchmark_runner/main/environment_variables.py)
+   1. Fill parameters: workload, kubeadmin_password, pin_node_benchmark_operator, pin_node1, pin_node2, elasticsearch, elasticsearch_port 
+   2. Run [main.py](/benchmark_runner/main/main.py)  and verify that the workload run correctly
+   3. The workload can be monitored and checked through 'current run' folder inside the run workload flavor (default flavor: 'test_ci')
+9. Open Kibana url and verify workload index populate with data:
+   1. Create the workload index: Kibana -> Hamburger tab -> Stack Management -> Index patterns -> Create index pattern -> workload-results -> timestamp -> Done
+   2. Verify workload-results index is populated: Kibana -> Hamburger tab -> Discover -> workload-results (index) -> verify that there is a new data
+
+## Add workload to grafana dashboard
```

Comment:
Which Grafana instance?

Reply:
Func or Perf environment Grafana instance

Reply:
At least specify that latter, then (and as much of a pointer as you can to where to find those).